subsys/event_manager/event_manager_shell.c

```diff
@@ -24,7 +24,7 @@ static int show_events(const struct shell *shell, size_t argc,
 		shell_fprintf(shell,
 			      SHELL_NORMAL,
 			      "%c %d:\t%s\n",
-			      (event_manager_displayed_events & BIT(ev_id)) ?
+			      (atomic_test_bit(event_manager_event_display_bm.flags, ev_id)) ?
```

Comment:
Shell uses preemptive context of low priority, maybe we could use spinlocks in this file to ensure that whole operations are done atomically ?

Reply:
would leave it as is